depthmapX/depthmapView.cpp

```diff
@@ -2472,8 +2474,12 @@ void QDepthmapView::OnEditSave()
        OutputSVG( stream, pDoc );

```

Comment:
Out of interest - what about SVGs? do they never have a scale?

Reply:
Nope, that was not implemented..
A comment in OuputSVG actually says:
   // This output SVG is a copy of the standard output... obviously, if you change
   // standard output, remember to change this one too!
   
   // also out of synch with EPS!

   // now the two are a little out of synch

Reply:
Hooray for quality code.